charts/rucio-server/templates/deployment.yaml

```diff
@@ -227,6 +264,10 @@ spec:
             - name: {{ $key1 | upper }}
               value: "{{ $val1  }}"
             {{- end}}
+            {{- if .Values.policyPackages.enabled}}
+            - name: PYTHONPATH
+              value: {{.Values.policyPackages.mountPath}}:${PYTHONPATH}
```

Comment:
I wonder if it would be cleaner to define the environmental variable in a similar fashion for the init container, as opposed to setting it in the snippet.

Reply:
I'd rather set it in the snippet to ensure it's always run before the python command